In [4]:
import pandas as pd

dataset = pd.read_csv("../2019 Winter Data Science Intern Challenge Data Set - Sheet1.csv")

dataset.head()

order_id  shop_id  user_id  order_amount  total_items payment_method  \
0         1       53      746           224            2           cash   
1         2       92      925            90            1           cash   
2         3       44      861           144            1           cash   
3         4       18      935           156            1    credit_card   
4         5       18      883           156            1    credit_card   

            created_at  
0  2017-03-13 12:36:56  
1  2017-03-03 17:38:52  
2   2017-03-14 4:23:56  
3  2017-03-26 12:43:37  
4   2017-03-01 4:35:11

In [5]:
dataset[["order_amount"]].describe()

order_amount
count    5000.000000
mean     3145.128000
std     41282.539349
min        90.000000
25%       163.000000
50%       284.000000
75%       390.000000
max    704000.000000

a. Our analysis shows that the average order values are mostly reasonable, but there is at least
one outlier in the data. There is an order for over 700,000 which is responsible for the
average order value being so high.

In [6]:
dataset[["order_amount"]].median()

order_amount    284.0
dtype: float64

b. I would report the median order value. This statistic is far more helpful in gaining a sense of where the majority
of the data lies. If allowed, I would also report the top of the first quartile and the top of the third quartile,
as they would give you a very good sense of the range of the data.

c. The median order value is 284 and the the top of the first quartile is 163 and the top of the third quartile is 390.